snapcraft/parts/lifecycle.py

```diff
@@ -23,17 +23,26 @@
 from dataclasses import dataclass
 from datetime import datetime
 from pathlib import Path
-from typing import TYPE_CHECKING, Any, Dict, List
+from typing import TYPE_CHECKING, Any, Dict, List, Tuple
 
 import craft_parts
 from craft_cli import EmitterMode, emit
-from craft_parts import ProjectInfo, StepInfo, callbacks, infos
+from craft_parts import ProjectInfo, StepInfo, callbacks
 
 from snapcraft import errors, extensions, pack, providers, utils
 from snapcraft.meta import manifest, snap_yaml
-from snapcraft.projects import GrammarAwareProject, Project
+from snapcraft.projects import (
+    Architecture,
+    ArchitectureProject,
+    GrammarAwareProject,
+    Project,
+)
 from snapcraft.providers import capture_logs_from_instance
-from snapcraft.utils import get_host_architecture, process_version
+from snapcraft.utils import (
+    ARCH_TRANSLATIONS_DEB_TO_PLATFORM,
```

Comment:
This is still importing the mapping directly, any special reason for not using the function here?

Reply:
Whoops!  I made the dictionaries private, added the function, and a note about 2 dictionaries.